doc/Options/z390_filetypes.md

```diff
@@ -0,0 +1,32 @@
+# z390 File types
+
+The scope value details where the option is used.
+
+* M - mz390 macro processor which reads ASCII assembler source and generates expanded BAL
+* A - az390 assembler which reads BAL and generated OBJ relocatible object code
+* L - lz390 linker which reads OBJ relocatible object files and generates 390 load module
+* E - ez390 emulator runtime which loads and executes 390 load module
```

Comment:
Line 6 and line 7. "relocatible" -> "relocatable".

Reply:
Resolved by 52df1c